src/V3Error.h

```diff
@@ -65,6 +65,7 @@ class V3ErrorCode final {
         E_TASKNSVAR,    // Error: Task I/O not simple
         E_NEEDTIMINGOPT,  // Error: --timing/--no-timing option not specified
         E_NOTIMING,     // Timing control encountered with --no-timing
+        E_LIFETIME,     // Error: Reference to a variable might outlive the variable.
```

Comment:
Move up to be sorted above E_LINT.  Don't forget to reorder strings below.

Reply:
I should have said above E_NEEDTIMINGOPT.  You are right they weren't sorted, please merge master as I sorted the sections.

Reply:
Done!